cpp/net/connection_pool.cc

```diff
@@ -30,20 +132,57 @@ const HostPortPair& ConnectionPool::Connection::other_end() const {
 
 
 ConnectionPool::ConnectionPool(libevent::Base* base)
-    : base_(CHECK_NOTNULL(base)), cleanup_scheduled_(false) {
+    : base_(CHECK_NOTNULL(base)),
+      cleanup_scheduled_(false),
+      ssl_ctx_(CHECK_NOTNULL(SSL_CTX_new(TLSv1_client_method())),
+               SSL_CTX_free) {
+  // Try to load trusted root certificates.
+  // TODO(alcutter): This is probably Linux specific, we'll need other sections
```

Comment:
I was kind of assuming here that the "Linux specific" bit (in the context that we're using OpenSSL, and not some other possibly platform-specific library) was the filename of where we got the trusted root certs.